src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/XmlDescriptor.php

```diff
@@ -373,6 +373,18 @@ private function getContainerDefinitionDocument(Definition $definition, $id = nu
         $serviceXML->setAttribute('autowired', $definition->isAutowired() ? 'true' : 'false');
         $serviceXML->setAttribute('file', $definition->getFile());
 
+        $autowiredCalls = array_filter($definition->getAutowiredCalls(), function ($method) {
+            return $method !== '__construct';
+        });
+        if ($autowiredCalls) {
+            $serviceXML->appendChild($autowiredMethodsXML = $dom->createElement('autowired-calls'));
```

Comment:
Me neither. But that's consistent with the way `tags`, `calls`, `properties`, ... are dumped. (See https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/definition_arguments_2.xml for instance)